scripts/run.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+import os
+
+from wikidatarefisland import data_access, Config, external_identifiers, services
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Run the pipes and/or side steps')
+    parser.add_argument('--step', default=[], dest='steps', action='append', type=str,
+                        help='Steps to run e.g. ss1 or pipe2, default: Run all')
+
+    steps = parser.parse_args().steps
+    if steps == []:
+        steps = ['all']
+
+    file_path = os.path.realpath(__file__)
+    config = Config.newFromScriptPath(file_path)
+    wdqs_reader = data_access.WdqsReader.newFromConfig(config)
+    storage = data_access.Storage.newFromScript(file_path)
+    external_identifier_formatter = services.WdqsExternalIdentifierFormatter(wdqs_reader)
+
+    if 'ss1' in steps or 'all' in steps:
```

Comment:
Looks good; once we start wiring up other steps I'd probably be keen to make this less "conditional" but let's go for this now.

Reply:
yeah